bugzilla/client.go

```diff
@@ -40,31 +40,20 @@ func (c *Client) addRequestHeaders(req *http.Request) {
 	}
 	// Bugzilla 5.0.4 and below don't support these headers
 	if len(c.APIKey) > 0 {
-		req.Header["X-Bugzilla-API-Key"] = []string{c.APIKey}
+		req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", c.APIKey))
 	}
 	if len(c.Token) > 0 {
 		req.Header["X-Bugzilla-Token"] = []string{c.Token}
 	}
 }
 
-func (c *Client) newRequestValues() url.Values {
-	v := make(url.Values)
-	if len(c.APIKey) > 0 {
-		v.Set("Bugzilla_api_key", c.APIKey)
-	}
-	if len(c.APIKey) > 0 {
-		v.Set("Bugzilla_token", c.Token)
```

Comment:
i wonder if this was even ever correct, or if it should have been checking len(c.Token)......oh well, guess it doesn't matter now.